src/main/kotlin/no/nav/syfo/application/service/VurderingService.kt

```diff
@@ -29,14 +29,28 @@ class VurderingService(
         gjelderFom: LocalDate?,
         callId: String,
     ): Vurdering {
-        val vurdering = Vurdering(
-            personident = personident,
-            veilederident = veilederident,
-            begrunnelse = begrunnelse,
-            document = document,
-            type = type,
-            gjelderFom = gjelderFom,
-        )
+        val vurdering = when (type) {
+            VurderingType.FORHANDSVARSEL -> Vurdering.Forhandsvarsel(
+                personident = personident,
+                veilederident = veilederident,
+                begrunnelse = begrunnelse,
+                document = document,
+            )
+            VurderingType.OPPFYLT -> Vurdering.Oppfylt(
+                personident = personident,
+                veilederident = veilederident,
+                begrunnelse = begrunnelse,
+                document = document,
+            )
+            VurderingType.AVSLAG -> Vurdering.Avslag(
+                personident = personident,
+                veilederident = veilederident,
+                begrunnelse = begrunnelse,
+                document = document,
+                gjelderFom = gjelderFom ?: throw IllegalArgumentException("gjelderFom is required for $type")
```

Comment:
Litt usikker på hvordan vi skal håndtere dette inntil vi har kode på plass i frontend for å sette `gjelderFom` :thinking: